src/main/java/tornadofx/LayoutDebugger.kt

```diff
@@ -406,7 +406,7 @@ class LayoutDebugger : Fragment() {
         override fun toString(v: Insets?) = if (v == null) "" else "${v.top.s()} ${v.right.s()} ${v.bottom.s()} ${v.left.s()}"
         override fun fromString(s: String?): Insets {
             try {
-                if (s == null || s.isEmpty()) return Insets.EMPTY
+                if (s.isNullOrBlank()) return Insets.EMPTY
```

Comment:
This is a semantic change. Did you mean `isNullOrEmpty()`?

Reply:
You are right it is a semantic change, but in this case, whether the string is empty or blank the result will be `Insets.EMPTY`, in the old code if the string was blank an exception was thrown and then ignored to return `Insets.EMPTY`. In the new code we prevent the exception to be thrown as we know that if it is black we will return `Insets.EMPTY`.

In my opinion this change is justified.